### PR TITLE
Scripts

### PR DESCRIPTION
**Pull request by princelySid**

### CODE REVIEW

File: scripts/06_create_coda_files.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# Check that the correct number of arguments were provided.
+if [ $# -ne 9 ]; then
+    echo "Usage: sh 06_create_coda_files.sh <user> <data-root> <json-input-path> <flow-name> <variable-name> <coding-scheme> <is-multi-coded> <has-yes-no> <auto-cleaner>"
```
Comment (as2388) [inline scripts/06_create_coda_files.sh]: As written, this file needs to be run multiple times with different configurations for each variable. I suggest changing this such that this file runs docker-run.sh repeatedly, with the flow-name, variable-name, coding-scheme etc. pre-configured for each run.

In other words, remove the variable-specific args and code them into repeated calls to docker-run.sh here.

Comment (princelySid) [reply scripts/06_create_coda_files.sh]: Changing all the scripts where this is applicable

File: README.md [hunk 2]
```diff
@@ -1,5 +1,127 @@
...
+- Bash
+
+Development requires the following additional tools:
+- Python 3.6+
+- pipenv
+- git
+## Usage
+Running the pipeline requires (0) creating a phone number <-> UUID table to support de-identification of respondents, (1) fetching the PDM survey and demographics from Rapid Pro, (2) fetching the contact data from Rapid Pro (3) merging the post distribution monitoring data and the demographics, (4) merging the outside data(SCOPE) to the rapidpro data, (5) creating files for ICR, (6) creating CODA files (7) merging the coded CODA data with the rest of the data, and (8) creating the analysis files.
+
+To simplify the configuration and execution of these stages, this project includes a `run_scripts` directory, which contains shell scripts for driving each of the stages. More detailed descriptions of the functions of each of those stages, and instructions for using the run scripts, are provided below.
```
Comment (as2388) [inline README.md]: You have called this `scripts` in the code

[the remark this answers is not in the document]
Comment (princelySid) [reply README.md]: Changed it

File: README.md [hunk 3]
```diff
@@ -1,5 +1,127 @@
...
+
+### 0. Phone Number <-> UUID Table
+First, create an empty phone number <-> UUID table by running the following command in the `run_scripts` directory:
+```
+$ ./1_create_uuid_table.sh <data-root>
```
[the remark this answers is not in the document]
Comment (princelySid) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -1,5 +1,127 @@
...
+
+To simplify the configuration and execution of these stages, this project includes a `run_scripts` directory, which contains shell scripts for driving each of the stages. More detailed descriptions of the functions of each of those stages, and instructions for using the run scripts, are provided below.
+
+### 0. Phone Number <-> UUID Table
+First, create an empty phone number <-> UUID table by running the following command in the `run_scripts` directory:
+```
+$ ./1_create_uuid_table.sh <data-root> 
+```
+
+where - `data-root` is an absolute path to the directory in which all pipeline data should be stored. The UUID table will be saved to a file in the directory <data-root>/UUIDs.
```
[the remark this answers is not in the document]
Comment (princelySid) [reply README.md]: Changed

File: scripts/07_merge_coda_files.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+echo "Merging Needs Met"
+./docker-run.sh "$USER" "$DATA_ROOT/07 Coded CODA with Messages/coded_coda_with_messages.json" \
+    "Needs_Met_Yesno" "$DATA_ROOT/coded_coda_files/needs_met_coded.json" \
+    "../coding_schemes/Needs_Met_Yesno.json" "$DATA_ROOT/07 Coded CODA with Messages/coded_coda_with_messages.json" \
+    "True" "True"
+
+echo "Merging Cash Modality"
+./docker-run.sh "$USER" "$DATA_ROOT/07 Coded CODA with Messages/coded_coda_with_messages.json" \
+    "$Cash_Modality_Yesno" "$$DATA_ROOT/coded_coda_files/cash_modality_coded.json" \
```
[the remark this answers is not in the document]
Comment (princelySid) [reply scripts/07_merge_coda_files.sh]: Yeah caught this when I was testing it. Fixed.

**Pull request merged**